test/binaryen.js/event.js

```diff
@@ -7,7 +7,7 @@ function cleanInfo(info) {
 }
 
 var module = new binaryen.Module();
-module.setFeatures(binaryen.Features.ExceptionHandling);
+module.setFeatures(binaryen.Features.ExceptionHandling | binaryen.Features.Multivalue);
```

Comment:
I made it so that multivalue events specifically require multivalue to be enabled in addition to exceptions. This file does have a multivalue event type, even though it is not used.